dev-db/postgresql/postgresql-10.23.ebuild

```diff
@@ -386,7 +386,14 @@ pkg_config() {
 	einfo "Initializing the database ..."
 
 	if [[ ${EUID} == 0 ]] ; then
-		su - postgres -c "${EROOT}/usr/$(get_libdir)/postgresql-${SLOT}/bin/initdb -D \"${DATA_DIR}\" ${PG_INITDB_OPTS}"
```

Comment:
For a change like this, you should probably revbump and put this into testing. Especially when you're changing the previous behavior.

Reply:
This does not change any behavior. The behavior is exactly the same, just using a different method of switching user by default.

Reply:
The behavior is not changing. The same command would be run, the only difference is the user switching mechanism. I pushed a change to retain the previous do-it-with-su-by-default.